resources/views/livewire/order/order-positions.blade.php

```diff
@@ -136,6 +145,29 @@ class="pb-4"
             </x-slot:footer>
         </x-card>
     </x-modal>
+    <x-modal name="add-discount">
+        <x-card>
+            <div class="flex flex-col gap-4">
+                <x-input
+                    prefix="%"
```

Comment:
why is it only possible to add a percentage discount this way?

Reply:
reverted this, i will create a new PR for the "bulk add"